soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -363,6 +372,9 @@ object SoQLFunctions {
     "Find the remainder(modulus) of one number divided by another"
   )
 
+  val NaturalLog = f("ln", FunctionName("ln"), Map("a" -> NumLike), Seq(VariableType("a")), Seq.empty, VariableType("a"))(
```

Comment:
Shall we add Log base 10 too?  I'll leave that up to you.

Reply:
For our purposes, we just need the `ln` function, but I have a few other SoQL function ideas on deck that I can add a `log(x)` function to as well!